### PR TITLE
Updated Search Bar 

### PR DESCRIPTION
**Pull request by TartejBrothers**
Fixing Issue #1263, updated the search bar

### CODE REVIEW

**Review by TartejBrothers — COMMENTED** [review]
Added Font Family

**Comment by daattali** [comment]
- Is the css for the placeholder necessary? I tried it without explicitly adding placeholder css, and it seems like the placeholder text still inherited the font from the input.
- What about the text of the results?

**Comment by TartejBrothers** [comment]
Ok, i will update that

**Review by TartejBrothers — COMMENTED** [review]
Removed Placeholder and added font family to result container anchor

**Comment by daattali** [comment]
Am I correct in my assessment though, or was the placeholder css used for a reason?

**Comment by TartejBrothers** [comment]
Yes, you are correct, its not required but I used just in case if it was inheriting font-family from somewhere else. Here in this case, thats not happening, so it will inherit input field font-family.

**Comment by daattali** [comment]
I don't think this will account for the case of "No results found". I think the css can be applied only to `#beautifuljekyll-search-overlay` and that will take care of everything

**Comment by TartejBrothers** [comment]
Hi, any updates on this

**Comment by TartejBrothers** [comment]
@daattali Any updates on this

File: assets/css/beautifuljekyll.css [hunk 1]
```diff
@@ -1003,6 +1003,7 @@ pre {
   text-align: center;
   padding: 1rem;
 }
+
```
Comment (PublisherApplications) [inline assets/css/beautifuljekyll.css:1006]: Thanks for adding thing.

File: assets/css/beautifuljekyll.css [hunk 2]
```diff
@@ -1030,7 +1031,9 @@ pre {
   outline: none;
   border: none;
   padding: 0 3rem;
+  font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (PublisherApplications) [inline assets/css/beautifuljekyll.css:1034]: Thanks for adding thing.

File: assets/css/beautifuljekyll.css [hunk 3]
```diff
@@ -1030,7 +1031,9 @@ pre {
...
 }
+
```
Comment (PublisherApplications) [inline assets/css/beautifuljekyll.css:1036]: Thanks for adding thing.

File: assets/css/beautifuljekyll.css [hunk 4]
```diff
@@ -1065,10 +1068,12 @@ pre {
   font-size: 1.5rem;
   max-height: calc(100vh - 6.5rem);
   overflow-y: auto;
+  font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (PublisherApplications) [inline assets/css/beautifuljekyll.css:1071]: Thanks for adding thing.

File: assets/css/beautifuljekyll.css [hunk 5]
```diff
@@ -1065,10 +1068,12 @@ pre {
...
 }
 #search-results-container a {
   color: #fff;
   text-decoration: none;
+  font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (PublisherApplications) [inline assets/css/beautifuljekyll.css:1076]: Thanks for adding thing.

**Review by PublisherApplications — COMMENTED** [review]
This changes are are approve

**Comment by TartejBrothers** [comment]
> > I think the css can be applied only to `#beautifuljekyll-search-overlay` and that will take care of everything
> 
> This would be a simpler fix, only one line should be added

I added that in the commit.

**Comment by daattali** [comment]
That's the **only** line that is required. No other lines need to change. Just one line is enough for the entire commit, no?

**Review by TartejBrothers — COMMENTED** [review]
Removed the extra font css too

**Pull request merged**